inc/fnlwfapi.h

```diff
@@ -73,6 +74,7 @@ FnLwfInitializeEa(
     EaHeader->EaValueLength = (USHORT)(EaLength - sizeof(*EaHeader) - sizeof(FNLWF_OPEN_PACKET_NAME));
 
     OpenPacket = (FNLWF_OPEN_PACKET *)(EaHeader->EaName + sizeof(FNLWF_OPEN_PACKET_NAME));
+    OpenPacket->ApiVersion = FNLWF_API_CURRENT_VERSION;
```

Comment:
Nothing actually reads it yet? I know there's only one version today, but to protect yourself from future high-version clients calling into you, shouldn't you check to make sure they are <= current?

Reply:
Yeah, we should, good point.